#422849: email customization improvements (#68)
bsi-software/bsi-cx-design-master-template-email: CHANGELOG.md

to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## Next

> * Add new variables in properties to handle the padding and the width of the footer and allignment for image and border for highlighted component

Reviewer: ```suggestion
* Add properties for padding and width of footer
* Add properties for alignment of image 
* Add properties for border of highlighted content
```
